x-pack/docs/en/rest-api/security/has-privileges.asciidoc

```diff
@@ -57,6 +65,13 @@ GET _xpack/security/user/_has_privileges
       "names": [ "inventory" ],
       "privileges" : [ "read", "write" ]
     }
+  ],
+  "application": [
```

Comment:
The intro for this example should be changed to say "...checks whether the current user has a specific set of cluster, index, and application privileges"

Reply:
Done!